src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure implements Figures {
+    private int side;
+
+    public Square() {
+        this.side = Main.rnd(1, 15);
+        this.area = area();
+        this.name = "square";
+        this.color = new ColorSupplier().getRandomColor();
+    }
+
+    public double area() {
+        return (double) this.side * this.side;
```

Comment:
```suggestion
        return (double) side * side;
```
якщо в тебе відсутня затінення змінних не потрібно писати ключове слово this